frontend/src/ui/Access.tsx

```diff
@@ -610,7 +625,7 @@ const formatUnknownRole = (role: string) => {
 
 /** Takes an initial ACL and formats it as options for react-select that are already selected. */
 const makeSelection = (record: AclRecord, acl: Acl): Option[] => {
-    const aclArray = [...new Set(acl.readRoles.concat(acl.writeRoles))];
+    const aclArray = [...new Set([...acl.readRoles, ...acl.writeRoles])];
```

Comment:
Uhg, the `Set` API is really limited :/  intersections and unions are normal set operations I would have expected. Sigh.